db-connect.js

```diff
@@ -0,0 +1,9 @@
+import mongoose from 'mongoose'
+const MONGO_URI = "mongodb://localhost/exercise_db";
+mongoose.connect(MONGO_URI, {
+    useNewUrlParser : true,
+    useUnifiedTopology : true,
+    useCreateIndex:true
+})
+.then(() => console.log('connected'))
+.catch(() => console.log('failed, err.message'))
```

Comment:
Slight issue here in the console.log. err.message is just printed as string.
So instead it would be `console.log('failed', err.message)`